BuilderDesignPattern/src/main/java/com/designPattern2/Director.java

```diff
@@ -0,0 +1,14 @@
+package com.designPattern2;
+
+public class Director {
+    public void constructBuilding(Builder builder) {
+
+    }
+
+    public void constructBuilding2(Builder builder) {

```

Comment:
What is the difference here between `constructBuilding` and `constructBuilding2`? The naming might be a bit confusing here.

Reply:
Sorry the building 2 should have been deleted